lib/Doctrine/Migrations/Metadata/Storage/TableMetadataStorage.php

```diff
@@ -198,7 +198,7 @@ private function isInitialized(): bool
         }
 
         if ($this->connection instanceof PrimaryReadReplicaConnection) {
-            $this->connection->connect('master');
+            $this->connection->connect();
```

Comment:
The equivalent of the old `->connect('master')` API of MasterSlaveConnection is `->ensureConnectedToPrimary()`, not `->connect()`